app/default/error_routes.py

```diff
@@ -21,26 +21,8 @@
 @account_bp.errorhandler(404)
 def not_found(error):
     current_app.logger.warning(f"Encountered 404 against url {request.path}: {error}")
```

Comment:
Can we drop this warning message as well? 🙏

Reply:
Deleted!